values/values.go

```diff
@@ -148,6 +148,74 @@ var (
 	Null = null{}
 )
 
+func Unwrap(v Value) (interface{}, error) {
```

Comment:
Can you remove the error from this and just document that this will panic when called with something that doesn't unwrap? We use that pattern in a lot of other areas so it shouldn't be a problem here.